src/wallet/rpcwallet.cpp

```diff
@@ -2836,20 +2836,20 @@ UniValue zc_raw_keygen(const UniValue& params, bool fHelp)
 
     auto k = SpendingKey::random();
     auto addr = k.address();
-    auto viewing_key = k.viewing_key();
+    auto receiving_key = k.receiving_key();
 
-    CDataStream viewing(SER_NETWORK, PROTOCOL_VERSION);
+    CDataStream receiving(SER_NETWORK, PROTOCOL_VERSION);
 
-    viewing << viewing_key;
+    receiving << receiving_key;
 
     CZCPaymentAddress pubaddr(addr);
     CZCSpendingKey spendingkey(k);
-    std::string viewing_hex = HexStr(viewing.begin(), viewing.end());
+    std::string receiving_hex = HexStr(receiving.begin(), receiving.end());
 
     UniValue result(UniValue::VOBJ);
     result.push_back(Pair("zcaddress", pubaddr.ToString()));
     result.push_back(Pair("zcsecretkey", spendingkey.ToString()));
-    result.push_back(Pair("zcviewingkey", viewing_hex));
+    result.push_back(Pair("zcviewingkey", receiving_hex));
```

Comment:
Hmm, the JSON key name is now inaccurate. This RPC method is deprecated but it still bugs me. Since the JSON key wasn't documented anyway, I suggest changing it to "zcreceivingkey" and adding it to the documented output.